networkit/test/test_distance_algorithms.py

```diff
@@ -12,55 +12,55 @@ def setUp(self):
 		self.L = nk.readGraph("input/looptest1.gml", nk.Format.GML) #without self-loops
 		self.LL = nk.readGraph("input/looptest2.gml", nk.Format.GML) #with self-loops sprinkled in
 
-	def testDistanceDiameter(self):
+	def testDiameter(self):
 		D = nk.distance.Diameter(self.LL, nk.distance.DiameterAlgo.ESTIMATED_RANGE, error = 0.1)
 		D.run()
 		D = nk.distance.Diameter(self.LL, nk.distance.DiameterAlgo.ESTIMATED_SAMPLES, nSamples = 5)
 		D.run()
 		D = nk.distance.Diameter(self.LL, nk.distance.DiameterAlgo.EXACT)
 		D.run()
+		self.assertIsInstance(D.getDiameter(),tuple)
```

Comment:
This only test the result of the last algorithm. I'd suggest splitting this test in 3 tests, one per algorithm.